src/portable/profile/strict/service.conf

```diff
@@ -23,6 +23,8 @@ NoNewPrivileges=yes
 MemoryDenyWriteExecute=yes
 RestrictRealtime=yes
 RestrictNamespaces=yes
+SystemCallFilter=@system-service
+SystemCallErrorNumber=EPERM
```

Comment:
I don't understand the comment about "strict" profile in the commit message. It seems to be changed the same as others.

Reply:
uh, that was supposed to say to exclude the "trusted" profile from this work. Will fix.